### DISCUSSION TITLE
What is the idiomatic way to split a BStr by the null character `\0`?

### DISCUSSION THREAD

**Discussion by kevinli1993** [BurntSushi/bstr#199] [Q&A] [answered]
Hi! 

It seems like `.split_str("\0")` could be used to split by null characters. I'm wondering if this is the recommended/idiomatic way to do this? (e.g. is there a `.split_byte(::u8)` that takes in a single byte?)

Thanks!

**Comment by BurntSushi** [comment, marked as answer]
Yes. That's idiomatic. I'm not aware of a reason why there should be a separate API for a single byte.

**Reply by kevinli1993** [reply]
Awesome, appreciate your work and the speedy response!
